pom.xml

```diff
@@ -4,8 +4,8 @@
     <parent>
         <groupId>gov.nsa.datawave.microservice</groupId>
         <artifactId>datawave-microservice-parent</artifactId>
-        <version>1.8</version>
-        <relativePath />
+        <version>2.0</version>
+        <relativePath>../../microservices/microservice-parent/pom.xml</relativePath>
```

Comment:
This works fine because 2.0 is a published release in our mvn repo.  The only time this doesn't work is if the version you are referring to is not in 1) your local maven repository, or 2) a remote maven repository